app/services/exactAlarmService.ts

```diff
@@ -0,0 +1,16 @@
+import { NativeModules } from 'react-native';
+
+const { ExactAlarmModule } = NativeModules;
+
+export const requestExactAlarmPermission = async (): Promise<boolean> => {
+  try {
+    // Request permission from the native module
+    const permissionGranted = await ExactAlarmModule.requestExactAlarmPermission();
+    // Return true if permission is granted, otherwise false
+    return permissionGranted;
+  } catch (error: any) {
+    console.log('Error requesting exact alarm permission:', error.message || error);    
```

Comment:
console.error for error messages will distinguish errors from other log messages.

Reply:
changes done as suggested